experiment/invitations-accepter/BUILD.bazel

```diff
@@ -0,0 +1,49 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+
+go_library(
```

Comment:
I think it would also be good to publish an image for this. @chaodaiG can we just update Bazel to publish another image or does that need additional setup somewhere?

Reply:
I think following the deleted part in https://github.com/kubernetes/test-infra/pull/20813/files might get it working

Reply:
The changes are required in `prow/BUILD.bazel`

Reply:
This tool doesn't live in prow's path. Do the changes in `prow/BUILD.bazel` can affect it?

Reply:
My understanding is that you:
* Need to build an image like here: https://github.com/kubernetes/test-infra/blob/master/prow/cmd/autobump/BUILD.bazel
* Change this to get it pushed in the postsubmit: https://github.com/kubernetes/test-infra/blob/master/prow/BUILD.bazel#L13

Reply:
Updated.